Indicators/EaseOfMovementValue.cs

```diff
@@ -0,0 +1,88 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+using QuantConnect.Data.Market;
+
+namespace QuantConnect.Indicators
+{
+    /// <summary>
+    /// This indicator computes the n-period Ease of Movement Value using the following:
+    /// MID = (high_1 + low_1)/2 - (high_0 + low_0)/2 
+    /// RATIO = (currentVolume/10000) / (high_1 - low_1)
+    /// EMV = MID/ratio
+    /// </summary>
+    public class EaseOfMovementValue : TradeBarIndicator, IIndicatorWarmUpPeriodProvider
+    {
+        private decimal _previousHighMaximum;
+        private decimal _previousLowMinimum;
+
+        /// <summary>
+        /// Gets a flag indicating when this indicator is ready and fully initialized
+        /// </summary>
+        public override bool IsReady => Samples >= 2;
+
+        /// <summary>
+        /// Required period, in data points, for the indicator to be ready and fully initialized.
+        /// </summary>
+        public int WarmUpPeriod => 2;
+
+        public override void Reset()
```

Comment:
nit - In Lean we usually place Constructor first. Missing xml docs

Reply:
Hi Martin, thank you for the feedback. The xml doc that is missing is QuantConnect.Indicators.xml?